donkeycar/tests/test_telemetry.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import time
+from unittest import mock
+from paho.mqtt.client import Client
+import donkeycar.templates.cfg_complete as cfg
+from donkeycar.parts.telemetry import Telemetry
+from unittest.mock import ANY
+
+def test_telemetry():
+
```

Comment:
Please avoid too many blank lines in functions  - pep-8.

Reply:
fixed